rcl/test/rcl/test_security_directory.cpp

```diff
@@ -18,13 +18,15 @@
 #include <algorithm>
 #include "rcl/security_directory.h"
 #include "rcutils/filesystem.h"
+#include "osrf_testing_tools_cpp/scope_exit.hpp"
+#include "rcl/error_handling.h"
 
 #define ROOT_NAMESPACE "/"
 #define TEST_SECURITY_DIRECTORY_RESOURCES_DIR_NAME "test_security_directory"
 #define TEST_NODE_NAME "dummy_node"
 #define TEST_NODE_NAMESPACE ROOT_NAMESPACE TEST_SECURITY_DIRECTORY_RESOURCES_DIR_NAME
 
-char g_envstring[512] = {0};
+char envstring[512] = {0};
```

Comment:
why the rename? I believe it's the ROS 2 style to prefix global vars with `g_*`.

Reply:
missed that. thanks for catching this! updated the name.